docs/kratos/self-service/flows/user-logout.mdx

```diff
@@ -3,115 +3,85 @@ id: user-logout
 title: User logout
 ---
 
-Ory Identities supports two logout flows:
+```mdx-code-block
+import { apiFlow } from "./code/logout"
+import CodeTabs from "@theme/Code/CodeTabs"
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import browserExample from "!!raw-loader!./code/logout/samples/browser/node.ts"
+import spaExample from "!!raw-loader!./code/logout/samples/spa/react.txt"
+import CodeBlock from "@theme/CodeBlock"
+```
 
-- Browser-based (easy): This flow works for all applications running on top of a browser. Websites, single-page apps,
-  Cordova/Ionic, and so on.
-- API-based (advanced): This flow works for native applications like iOS (Swift), Android (Java), Microsoft (.NET), React Native,
-  Electron, and others.
+The logout flow is used to terminate an Ory Identities session. Ory supports logout on all supported clients:
 
-import { apiFlow, browserFlow } from "./code/logout"
-import CodeTabs from "@site/src/theme/Code/CodeTabs"
+- [Server rendered browser applications](#server-side-browser-applications)
+- [Client side browser applications](#client-side-browser-applications)
+- [Native applications](#native-api-applications)
 
-## Self-service logout for server-side browser applications
+## Server-side browser applications
 
-To ensure that a logout was indeed intended by the user, Ory Identities first generates a Logout URL for a given Ory Session
-Cookie. You can then open the Logout URL in the Browser.
+To ensure that a logout was intended by the user, Ory Identities first generates a logout URL for a given Ory Session cookie. You
+can then open the logout URL in the browser.
 
-import SelfServiceLogoutBrowserFlow from "@site/src/theme/SelfServiceLogoutBrowserFlow"
+The logout URL is valid for the duration of the session. This means that you can call the endpoint once and cache this URL for
+some time before you call the endpoint again.
 
-<SelfServiceLogoutBrowserFlow />
+Alternatively, you can create the logout URL only when the user actually clicks the logout button and immediately call the
+resulting `logout_url` from your server to log out the user. Beware, that this method can lead to inconsistent behavior for users
+that switch accounts often.
```

Comment:
```suggestion
Alternatively, you can call the endpoint at the moment when the user performs the action required to log out 
and immediately call the `logout_url` from your server to log out the user. 

:::warning

This approach can lead to inconsistent system behavior for users who often switch between accounts

:::
```

So I edited this but I can't say I understand how this is different (for the end user) from the other behavior. I mean, it still logs me out when I click a "log out" button, right? How can this second behavior confuse me in any way? Isn't this just a backend difference? The first approach requires implementing some caching mechanism while the other doesn't?

Reply:
So two different accounts (A + B) use the same cookie? If that's the case then what you said makes sense to me. But I'd except a cookie per identity and then I don't know how this would work. 🤔

Reply:
You only have one cookie per browser. Tabs share all cookies for a page. So yes, effectively: two accounts use the same cookie.

Reply:
Oh, okay. I'll come up with a way to describe it better then.